STYLE_GUIDE.md

```diff
@@ -154,6 +154,27 @@ Prefer:
 
 ## JavaScript
 
+### Variable naming
+
+- Prefer camelCase over snake_case, except in JSON.
+- Try to use verbs for function/method names, and nouns for constant values.
+- Boolean variable names should give some indication of their state.
+- Prefer clarity over brevity, and avoid single-letter names and abbreviations unless they are well known for this domain (e.g. JSON).
+
+Avoid:
```

Comment:
I know it will get a bit long but I think examples like these are probably needed directly after each suggestion ideally